ServerCore/ServerCore.csproj

```diff
@@ -54,6 +54,9 @@
     <Content Update="Pages\Events\AddAdminOrAuthor.cshtml">
       <Pack>$(IncludeRazorContentInPack)</Pack>
     </Content>
+    <Content Update="Pages\Puzzles\Checklist.cshtml">
+      <Pack>$(IncludeRazorContentInPack)</Pack>
```

Comment:
What is this section anyway?

Reply:
Not sure - it was auto-generated so I didn't touch it